docs/usage/getting-started/use-cases.md

```diff
@@ -0,0 +1,135 @@
+Renovate considers these all to be forms of "package managers" and "package files" and therefore detects and updates them accordingly.
+
+Docker-compatible images are one of the key building blocks of modern software and so are most commonly found, in both CI/CD pipeline configs as well as referenced in IaC files.
+Renovate will detect these then look up tags and digests from Docker registries to determine not only if a new tag exists but also if a new digest does too.
+
+An example of tag-based updating would be `node` images from Docker Hub.
+These are typically tagged using their version, like `14.17.4` but can also have more elaborate tags like `14.17.4-alpine3.11`.
+Renovate handles both these tag scenarios and will propose updates such as from `14.17.4` to `14.7.5`, or from `14.17.4-alpine3.11` to `14.17.5-alpine3.11`.
+
+A better example of Renovate's power is when updating Docker digests.
+While humans could be reasonably expected to check and update versions like `14.17.4`, looking up image digests and updating them manually is impractical to do at scale.
+Renovate can not only keep Docker digests updated, but it can even be configured to "pin" digests from being tag-based to being tag+digest based to get immutable builds.
+
+### Internal Package Updates
+
+Companies typically have at least dozens of repositories, if not hundreds or thousands.
+In most cases, these repositories do not operate in isolation and may have upstream or downstream internal dependencies.
+In such cases, it is best practice to:
+
+- Update downstream links as soon as possible, and
+- Keep internal version use as consistent as possible
+
+Renovate is often used to achieve both the above best practices by detecting and updating internal dependencies just like external or Open Source dependencies.
+
+An example from Renovate itself is the use of submodule updating to automate the process of updating Renovate's documentation:
+
+- Renovate's main repository [`renovatebot/renovate`](https://github.com/renovatebot/renovate) contains the majority of Markdown documentation files
+- Renovate's documentation build repository [`renovatebot/renovatebot.github.io`](https://github.com/renovatebot/renovatebot.github.io) contains a submodule link to `renovatebot/renovate`
+- Submodule updates are performed automatically whenever detected
+- After the automatic update is merged, the documentation site is rebuilt and pushed live
+
+The above use case makes use of Renovate's "automerge" feature, which allows for fully automated updates without need for manual approval, merging, or even a PR at all if desired.
+Automerge is particularly useful for internal dependencies when it's best to use the approach of "if it passes tests then merge it".
+
+## Advanced Configuration Use
```

Comment:
```suggestion
## Advanced configuration
```

Maybe we're better of naming this like my suggestion? The word `use` seems a bit odd here.

I'd expect either:

- No `use` in header at all _or_
- `Advanced configuration use cases`